etc/docker/init-vnext-core.sh

```
@@ -0,0 +1,576 @@
+        echo "  Raw response: $response"
+        return 1
+    fi
+}
+
+# Function to merge custom components with core sys-* file
+merge_custom_components() {
+    # Redirect all output to stderr except the final result
+    {
+    local sys_file="$1"
+    local custom_folder="$2"
+    local folder_name="$3"
```

Shellcheck (0.11.0)

[warning] folder_name appears unused. Verify use (or export if used externally).

(SC2034)